server/checkins/checkinModel.js

```diff
@@ -0,0 +1,29 @@
+var mongoose = require('mongoose');
+
+var CheckinSchema = new mongoose.Schema({
+  queueHero: {
+    type: String,
+    required: true, 
+  }, 
+  vendor: {
+    type: String, 
+    required: true, 
+  },
+  meetingLocation: {
+    type: "Point",
+    coordinates: [ <longitude>, <latitude> ], 
```

Comment:
No space in after opening bracket and before closing bracket.

`[<longitude>, <latitude>`